includes/utilities.inc

```diff
@@ -87,20 +88,32 @@ function islandora_scholar_get_view(AbstractObject $object) {
     );
   }
   elseif (isset($object['PREVIEW']) && islandora_datastream_access(ISLANDORA_VIEW_OBJECTS, $object['PREVIEW'])) {
-    $image = theme('image', array(
+    $image = theme(
+      'image',
+      array(
         'path' => "islandora/object/$object->id/datastream/PREVIEW/view",
-      ));
+        'alt' => t('PREVIEW Datastream'),
+      )
+    );
     if (isset($object['PDF']) && islandora_datastream_access(ISLANDORA_VIEW_OBJECTS, $object['PDF'])) {
       $display['preview'] = array(
-        '#markup' => l($image, "islandora/object/$object->id/datastream/PDF/view", array(
+        '#markup' => l($image, "islandora/object/$object->id/datastream/PDF/view",
+          array(
             'html' => TRUE,
-          )),
+            'attributes' => array(
+              'alt' => t('PDF Datastream'),
```

Comment:
Just curiosity. Is having an "alt" for the image and also for the link that wraps the image a requirement for WCAG 2.0? when you put your cursor over the image on a rendered webpage, which one is displayed?

Reply:
This was my mistake, the alt should just be used for the image, and title used for the parent link.